### DISCUSSION TITLE
My school is not listed on the drop down list

### DISCUSSION THREAD

**Discussion by codechux** [community/community#141456] [GitHub Education]
### Hi everyone,

I would like to submit an application as a student for the Github education but I can't find my school name there on the list.
how do I go around this?

**Comment by mostaphabr** [comment]
![Screenshot 2024-10-14 153258](https://github.com/user-attachments/assets/8172d13c-e83e-4c49-9471-b0edb38bd390)
Make sure to link your GitHub account to the email address provided by your school. If my response was helpful, please mark it as the accepted answer.

**Reply by codechux** [reply]
my school did not issue me an email
is there another way?

**Reply by mostaphabr** [reply]
If your school hasn't issued you an email, GitHub may still allow you to apply for the Student Pack by verifying your student status through other means, such as a school ID or enrollment letter. However, having a verified school email usually simplifies the process .if my response was helpful, please mark it as the accepted answer.

**Reply by mostaphabr** [reply]
I recommend that you visit these community discussions; you will find solutions for your problem.

https://github.com/orgs/community/discussions/111352#user-content-how-long-after-ive-been-approved-will-i-receive-my-academic-benefits
